### ISSUE TITLE
background-position problem

### ISSUE THREAD

**Issue by jtenner** [JacksonTian/spritemate#2]
My generated sprite image is 7252px high, but upon further inspection is actually sets the sprite position to the following css.

```
Aya_smirk { width: '304px',
  height: '605px',
  'background-position': '0 -7252px',
  backgroundImage: 'url(\\sprites_56a7959790.png)' }
```

That's impossible!

The height itself is 7252px, so how could the upper-left be set at 7252?

I can actually fix this myself by using absurd.js and modifying the background-position values myself.

Please fix!

**Comment by jtenner** [comment]
```
app.use(function (next) {
    var ctx = this;
    var height = 0;
    for (var i = 0; i < this.images.length; i++) {
      var png = this.images[i];
      png.bitblt(this.out, 0, 0, png.width, png.height, 0, height);
      
      // add css output
      this.styles[images[i]] = {
        width: png.width + 'px',
        height: png.height + 'px',
        'background-position': '0 ' + (height > 0 ? ('-' + height + 'px') : '0')
      };
      // this should fix your bug
      height += png.height;
    }
    var pool = new Pool();
    this.out.pack().pipe(pool);
    pool.on('finish', function () {
      ctx.pool = pool;
      ctx.md5 = md5(pool.buffers);
      next();
    });
  });
```

**Comment by JacksonTian** [comment]
closed via #3

**Comment by jtenner** [comment]
@JacksonTian Thank you very much!
